bondarilya/second/task1.py

```diff
@@ -0,0 +1,31 @@
+"""
+Вхідні дані: текст довільної довжини, який може містити літери латинського алфавіту, пробіли та розділові знаки (,.:;!?-);
+Результат роботи: список слів (у нижньому регістрі), які зустрічаються в тексті найменшу кількість раз.
+Слова, записані через дефіс, вважати двома словами (наприклад, "hand-made"). Слова у різних відмінках, числах та з іншими перетвореннями (наприклад, "page" та "pages") вважаються різними словами. Регістр слів -- навпаки, не має значення: слова "page" та "Page" вважаються 1 словом.
+
+Якщо слів, які зустрічаються найменше, декілька -- вивести їх в алфавітному порядку.
+Приклад: 'Hello,Hello, my dear!'
+Результат: [my,dear]
+
+"""
+import re
+re_str = re.compile("^\w*\,\.\:\;\!\?\-+$")
```

Comment:
This regular expression is written uncorrectly. If you wanted to use it for finding words, you better use "^[\w-]+$"

![skrin2](https://user-images.githubusercontent.com/55147091/71354488-c5a42080-2584-11ea-96ee-e69fc246d1ac.png)
![skrin](https://user-images.githubusercontent.com/55147091/71354433-9ee5ea00-2584-11ea-8144-573869844ad9.png)